src/plugins/auto_batch/tests/unit/plugin_get_property_test.cpp

```diff
@@ -63,6 +64,8 @@ const std::vector<get_property_params> get_property_params_test = {
     get_property_params{ov::device::priorities.name(), true},
     get_property_params{ov::cache_dir.name(), true},
     get_property_params{ov::hint::performance_mode.name(), true},
+    get_property_params{ov::enable_profiling.name(), false},
+
```

Comment:
Please remove blank line to solve CI issue:
![image](https://github.com/openvinotoolkit/openvino/assets/31196718/ac0f97fc-70f6-429a-bad3-d830c0c078c8)